docs/throttling.md

```diff
@@ -0,0 +1,59 @@
+
+## The 3G network throttling preset
+
+This 3G preset is the standard recommendation:
+
+- Latency: 150ms
+- Throughput: 1.6Mbps down / 750 Kbps up.
+- Packet loss: none.
+
+These exact figures are used as the [WebPageTest "Mobile 3G - Fast" preset](https://github.com/WPO-Foundation/webpagetest/blob/master/www/settings/connectivity.ini.sample) and [Lighthouse's throttling default](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/emulation.js).
+
+## Throttling basics
+
+1. The DevTools network throttling, which Lighthouse uses, is implemented within Chrome at the request-level. As a result, it has some downsides that are now summarized in this doc: [Network Throttling & Chrome - status](https://docs.google.com/document/d/1TwWLaLAfnBfbk5_ZzpGXegPapCIfyzT4MWuZgspKUAQ/edit). The TLDR: while it's a [decent approximation](https://docs.google.com/document/d/1uS9SH1KpVH31JAmf-iIZ61VazwAF9MrCVwETshBC4UQ/edit), it inaccurately models a slow connection.
```

Comment:
> inaccurately models a slow connection

would it be more correct to say it's not a sufficient model of a slow connection? At least to me "inaccurately models" makes it sound more like you shouldn't use it at all rather than treating it as an approximation that doesn't handle several aspects of a slow connection.